### ISSUE TITLE
Errors in generating PDF

### ISSUE THREAD

**Issue by Anubarak** [craftcms/commerce#1081]
### Description
You have 2 fatal errors here https://github.com/craftcms/commerce/blob/develop/src/services/Pdf.php#L134

```PHP
        // Trigger an 'afterRenderPdf' event
        $afterEvent = new PdfEvent([
            'order' => $event->order,
            'option' => $event->option,
            'template' => $event->templatePath,
            'variables' => $variables,
            'pdf' => $dompdf->output(),
        ]);
        $this->trigger(self::EVENT_AFTER_RENDER_PDF, $afterEvent);
        return $event->pdf;
```

https://github.com/craftcms/commerce/blob/develop/src/events/PdfEvent.php#L37 only is

```PHP
    /**
     * @var string
     */
    public $template;
```

`$template` instad of `$event->templatePath` 

`return $event->pdf;` must be `return $afterEvent->pdf;`
The correct code is
```PHP
        // Trigger an 'afterRenderPdf' event
        $afterEvent = new PdfEvent([
            'order' => $event->order,
            'option' => $event->option,
            'template' => $event->template,  // fixed template
            'variables' => $variables,
            'pdf' => $dompdf->output(),
        ]);
        $this->trigger(self::EVENT_AFTER_RENDER_PDF, $afterEvent);

        return $afterEvent->pdf; // return the correct one
```

### Additional info

- Commerce: 2.2.4

**Comment by Mosnar** [comment]
I opened a PR for this a couple days ago: #1072

**Comment by MaximilianSchwarz** [comment]
I also have this issue  :)

**Comment by lukeholder** [comment]
Fixed in merge.

**Comment by lukeholder** [comment]
Released in 2.2.6
